main/src/main/scala/org/clulab/processors/Document.scala

```diff
@@ -24,12 +31,15 @@ class Document(val sentences: Array[Sentence]) extends Serializable {
   /** The original text corresponding to this document, if it was preserved by the corresponding processor */
   var text: Option[String] = None
 
+  /** Map of any arbitrary document attachments such as document creation time */
+  protected var attachments: Option[mutable.HashMap[String, DocumentAttachment]] = None
```

Comment:
Maybe this is just processors style, but why use an `Option` to mean "no attachments", rather than just an empty `Map`?

Reply:
I blame those mathematicians who are always distinguishing between the null set and the empty set.  I think it is to follow processors style though.